include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -283,6 +283,10 @@ SECTIONS
 	}
 #endif
 
+#if defined(CONFIG_NOCACHE_MEMORY)
+#include <linker/ram-nocache.ld>
```

Comment:
I wonder: since you provide a solution for ARM only, should this file be hosted under ./arm/....?

Reply:
I provide a solution for ARM only for now, but the linker file can work for other architectures provided the default alignment is enough.